docs/internal/Level-1+-Reviews-and-Changes-Requested.md

```diff
@@ -0,0 +1,131 @@
+  - in the list of applications the action `CONTINUE_REVIEW` is available for this user now
+* On review level 2 **submission**:
+  - trim `review_response` that have been created but not reviewed by user
+  - update other `review_response` status to `SUBMITTED`
+  - update `review` status to `SUBMITTED`
+  - update reviews with associated `review_response` (by `review_response_link_id`) that have decision of `disagree` by level 2 reviewer with status to `CHANGES_REQUESTED`
+  - in the list of applications the action `VIEW_REVIEW` is available for this user now (review is not editable)
+  - in the list of applications the action `UPDATE_REVIEW` is available for users owner of the Review with status `CHANGES_REQUESTED`
+* On review level 1 **update**:
+  - creates new `review` with status as `DRAFT`
+  - create `review_response` with status `DRAFT` duplicating lastest `review_response` with status `SUBMITTED` from this user
+  - in the list of applications the action `CONTINUE_REVIEW` is available for this user now
+* On review level 1 **submission**:
+  - trim `review_responses` that are duplicated and not changed
+  - update `review` status to `SUBMITTED`
+  - update other `review` of level 2 with `review_question_assignment` associated to `review_response` status to `PENDING`
+  - in the list of applications the action `VIEW_REVIEW` is available for this user now (review is not editable)
+  - in the list of applications the action `RESTART_REVIEW` is available for users owner of the Review with status `PENDING`
+
+### Front End
+
+- Fetches actions and calculates progress of review (from application) or review with changes requested
+- GraphQL mutation to create review responses - linked to another review or to application response
+- GraphQL mutation to update `review_response` and `review_decison`
+- Validates submission of review and give options to submmit based on `is_last_level` and if review is level 1+
+- GraphQL mutation to request submission of review
+
+Similart to what was in previous [Level 1 Reviews](Level-1-Review-And-LOQ.md), but considering review of review.
+
+## Validation of Submission
+
+### Review
+
+First review validation in more details on [Level 1 Review and LOQ](Level-1-Review-And-LOQ.md).
+
+When **review** is updated as a result of `Changes Requested` by a Level 1+ reviewer, it can only be submitted if general validation passes AND all questions with change requests have been changed.
+
+Logic to check validation of a **changes-requested review** is:
+
+- `thisReviewLastestResponse` is different to `thisReviewPreviousResponse` that has some associated `reviewResponse.reviewResponseLinkId` with decision of `disagree`. This is more easily validated with fields `isChangeRequest` and `isChanged` in the 'state' of each element.
+
+### Consolidation
+
+Logic to check validation is:
+
+- On `disagreeement` it will configure a **changes-quested** decision and submitted back to other reviewers
```

Comment:
```suggestion
- On `disagreement` it will configure a **changes-requested** decision and be submitted back to other reviewers
```
Also -- is "configure" the right term here? Could it be "create"?

Reply:
Yes, if that's clearer. I'll change now